### PR TITLE
Moving place type workspaces to public beta

### PR DESCRIPTION
**Pull request by slister1001**
Updating place, room, roomList, and workspace related documentation to help us bring workspace documentation to public beta.

### CODE REVIEW

File: changelog/Microsoft.Exchange.Places.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+﻿{
+  "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "f685e2e7-e767-4dc2-b668-3516e9ccf8d5",
+          "ApiChange": "Resource",
+          "ChangedApiName": "spaces",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **spaces** relationship from [roomList](https://docs.microsoft.com/en-us/graph/api/resources/roomList?view=graph-rest-beta) resource.",
```
Comment (angelgolfer-ms) [inline changelog/Microsoft.Exchange.Places.json:10]: Lines 5-12:
The current public beta metadata doesn't have a spaces nav property in roomList, and the published docs don't have that either.
Was this changelist record generated by Graph Studio?
Either way, from the customers' perspective, this doesn't apply. Please remove these lines.

File: api-reference/beta/resources/roomlist.md [hunk 2]
```diff
@@ -13,16 +13,17 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Represents a group of [room](room.md) objects defined in the tenant.
+Represents a group of [room](room.md) and [workspace](workspace.md) objects defined in the tenant.
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/roomlist.md:16]: Can a roomlist include a combination of rooms and workspace resources? 
A roomlist doesn't have to be homogenous (only room resources or only workspace resources), is that right?

File: api-reference/beta/resources/place.md [hunk 3]
```diff
@@ -13,23 +13,12 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Represents basic location attributes such as name, physical address, and geographic coordinates. This is the base type for richer location types such as [room](room.md) and [roomList](roomlist.md).
+Represents basic location attributes such as name, physical address, and geographic coordinates. This is the base type for richer location types such as [room](room.md), [workspace](workspace.md), and [roomList](roomlist.md).
 
 ### Using the places API
-Exchange Online administrators can organize meeting rooms in a tenant into room lists. Using the places API, you can get all the room lists or rooms in the tenant, or get all the rooms in a specific room list.
+Exchange Online administrators can organize either meeting rooms or work spaces in a tenant into room lists. Using the places API, you can get all the room lists, rooms, or work spaces in the tenant, or get all the rooms/work spaces in a specific room list.
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/place.md:19]: Line 19:
The "either... or" indicates that a roomlist can have only rooms, or only work spaces, but not a mix. Can you confirm please?

File: api-reference/beta/resources/place.md [hunk 4]
```diff
@@ -13,23 +13,12 @@ Namespace: microsoft.graph
...
 
-Places like [room](room.md) and [roomList](roomlist.md) contain the basic **id**, display name and email address. In addition, they contain navigational information like physical address and geographical coordinates, and in the case of rooms, other relevant information such as AV capabilities, floor number, and capacity.
-
-The [findRooms](../api/user-findrooms.md) and [findRoomLists](../api/user-findroomlists.md) functions support similar lookup for rooms and room lists in a tenant. The following is a comparison between the places API and these functions.  If you are creating a production app, choose the places API as the API is now generally available in v1.0. Plan to update any existing code that uses **findRooms** or **findRoomLists** to use the places API, because **findRooms** or **findRoomLists** will be deprecated, and a timeline will be announced.
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/place.md:23]: Lines 23-32:
Regarding findRooms and findRoomLists, they are still in the beta metadata and their docs still present them as beta API. So from the customers' perspective, nothing has changed since they got into beta.
Is there a better defined plan and timeline for their deprecation?

Meanwhile, without updating their docs, customers may still perceive them as a choice for similar use cases. We should keep the comparison section here until we're well into messaging their deprecation.

**Review by angelgolfer-ms — COMMENTED** [review]
Hi @slister1001 , I was responding to your internal ping on Teams and clarified what we need to prepare for a content review. I quickly browsed through your PR anyway and added a few comments.
Can you please provide what's needed per my Teams message for a proper content review?
Also, it would be helpful if you make clarifications per my comments in the PR.

Thanks.

File: changelog/Microsoft.Exchange.Places.json (hunk 1 above)
Comment (slister1001) [reply changelog/Microsoft.Exchange.Places.json:10]: Yes, it was generated by Graph Studio based on changes made on my branch in the Workloads repo. I will remove these lines regardless.

File: api-reference/beta/resources/place.md (hunk 3 above)
Comment (slister1001) [reply api-reference/beta/resources/place.md:19]: A roomlist can have both rooms and workspaces, I will update this line to reflect that.

File: api-reference/beta/resources/place.md (hunk 4 above)
Comment (slister1001) [reply api-reference/beta/resources/place.md:23]: I will keep the comparison section included in here. We are discussing a more concrete timeline for deprecation of these APIs, it has been in our backlog for some time now.

File: api-reference/beta/resources/place.md [hunk 5]
```diff
@@ -13,14 +13,14 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Represents basic location attributes such as name, physical address, and geographic coordinates. This is the base type for richer location types such as [room](room.md) and [roomList](roomlist.md).
+Represents basic location attributes such as name, physical address, and geographic coordinates. This is the base type for richer location types such as [room](room.md), [workspace](workspace.md), and [roomList](roomlist.md).
 
 ### Using the places API
-Exchange Online administrators can organize meeting rooms in a tenant into room lists. Using the places API, you can get all the room lists or rooms in the tenant, or get all the rooms in a specific room list.
+Exchange Online administrators can organize meeting rooms and work spaces in a tenant into room lists. Using the places API, you can get all the room lists, rooms, or work spaces in the tenant, or get all the rooms or work spaces in a specific room list.
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/place.md:19]: Line 19 and GLOBAL:
I just verified, and FYI, per Microsoft Writing Style Guide, when using the words "work" and "space" together in natural language, please use the term "workspace" with no space between them. 
See https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36451.

Related: when referring to the resource name, it's also "workspace", one word, except that you would apply a link to that topic workspace.md on first reference, or in subsequent references, style the word bold.

Comment (angelgolfer-ms) [reply api-reference/beta/resources/place.md:19]: FYI I searched and replaced the use of "work space" (two words) in natural language in your PR by "workspace" (one word).
For a more consistent learning experience for customers, I also replaced the use of "work space" (two words) as values for displayName and nickname in your examples.

File: changelog/Microsoft.Exchange.Places.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+﻿{
+  "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "f685e2e7-e767-4dc2-b668-3516e9ccf8d5",
+          "ApiChange": "Resource",
+          "ChangedApiName": "workspaces",
```
Comment (angelgolfer-ms) [inline changelog/Microsoft.Exchange.Places.json:8]: Line 7:
Since this changelist record is about adding a nav property, ApiChange should be "Relationship".
See example 3 in the [changelog ](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-examples) topic in the onboarding docs.

Comment (angelgolfer-ms) [reply changelog/Microsoft.Exchange.Places.json:8]: Can you confirm if line 7 was what's generated by Graph Studio?
If that's the case, that's a bug we need to file for Graph Studio.

File: api-reference/beta/resources/roomlist.md (hunk 2 above)
Comment (angelgolfer-ms) [reply api-reference/beta/resources/roomlist.md:16]: Like this:
"Represents a group of [room](room.md) or [workspace](workspace.md) resources defined in the tenant. A **roomList** can contain a mix of **room** and **workspace** resources. " 
Use "or" in the first sentence.  Then add a sentence to state a roomList can contain both types.

Comment (angelgolfer-ms) [reply api-reference/beta/resources/roomlist.md:16]: FYI to reduce the churn, I updated line 16 on your behalf.

File: api-reference/beta/api/place-list.md [hunk 7]
```diff
@@ -13,19 +13,18 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Get a collection of the specified type of [place](../resources/place.md) objects defined in the tenant. For example, you can get all the rooms, all the room lists, or the rooms in a specific room list in the tenant.
+Get a collection of the specified type of [place](../resources/place.md) objects defined in the tenant. For example, you can get all the rooms, all the workspaces, all the room lists, the workspaces in a specific room list,  or the rooms in a specific room list in the tenant.
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/place-list.md:17]: Line 16:
The "for example" sentence has gotten a bit hard to parse; usually a sentence to give examples should include no more than 3 examples. In this context, I think it's worthwhile to list what is actually supported (rather than saying "for example").

Comment (angelgolfer-ms) [reply api-reference/beta/api/place-list.md:17]: FYI I included a bullet list which basically reflects the examples in this topic, and links to each example for convenience for the customers. The bullet list would be easier to scan, and the links to examples can promote learning.

File: api-reference/beta/api/place-list.md [hunk 8]
```diff
@@ -13,19 +13,18 @@ Namespace: microsoft.graph
...
 
 A **place** object can be one of the following types:
 
 * A [room](../resources/room.md) which includes rich properties such as an email address for the room, and accessibility, capacity, and device support. 
+* A [workspace](../resources/workspace.md) which includes properties such as an email address for the workspace, and accessibility and capacity. 
 * A [room list](../resources/roomlist.md) which includes an email address for the room list, and a navigation property to get the collection of room instances in the room list. 
 
-Both **room** and **roomList** are derived from the **place** object.
+All **room**, **workspace** and **roomList** are derived from the **place** object.
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/place-list.md:24]: Line 24:
FYI I removed the "all". "All" can be ambiguous for Loc in this context. We simply want to say, each of those resources is derived from place.

File: api-reference/beta/api/place-list.md [hunk 9]
```diff
@@ -13,19 +13,18 @@ Namespace: microsoft.graph
...
 
 By default, this operation returns 100 places per page. 
 
-Compared with the [findRooms](../api/user-findrooms.md) and [findRoomLists](../api/user-findroomlists.md) functions, this operation returns a richer payload for rooms and room lists. See [details](../resources/place.md#using-the-places-api) for how they compare.
-
 ## Permissions
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/place-list.md:28]: FYI, I am adding back the sentence you deleted above back to the place-list.md topic, for reasons described in my previous pass:

"Compared with the [findRooms](../api/user-findrooms.md) and [findRoomLists](../api/user-findroomlists.md) functions, this operation returns a richer payload for rooms and room lists. See [details](../resources/place.md#using-the-places-api) for how they compare."

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
@slister1001 , I went ahead to go over your incorporation last week, thanks for doing that.
I also made some changes directly to your PR. Please feel free to take a look at my commit and let me know if you notice any issues or have any questions,

In the interest of time (as we're so close to Build), this time I reviewed your PR without first confirming what's in the api.md and CSDL files. That's usually not a reliable practice. Once you specify their location in an internal roadmap item, I'll take a look to make sure my understanding is consistent with those files. Please ping me on Teams once you have the roadmap item ready.

**Review by angelgolfer-ms — APPROVED** [review]
Hi @slister1001 ,
On you behalf, I addressed this [comment ](https://github.com/microsoftgraph/microsoft-graph-docs/pull/16504/files#r868872246) that I had left 2 days ago, so that we can wrap up this PR.
On your end, I still need you to confirm this other [comment ](https://github.com/microsoftgraph/microsoft-graph-docs/pull/16504/files#r868875500)to see if we need to file a bug for Graph Studio.



And after you do so, when your API is available to customers, you should please follow step 8 of the [content workflow](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211773/Content-workflow?anchor=workflow-for-internal-contributors-to-add-new-content-or-update-content), so that the docs can be published as well.

File: changelog/Microsoft.Exchange.Places.json (hunk 6 above)
Comment (slister1001) [reply changelog/Microsoft.Exchange.Places.json:8]: Yes, I can confirm that line 7 and the entirety of the changelog was generated by Graph Studio

File: api-reference/beta/api/place-list.md [hunk 10]
```diff
@@ -212,13 +230,119 @@ Content-type: application/json
...
 }
 ```
+### Example 2: List all the workspaces defined in the tenant
+
+#### Request
+
+The following example shows how to get all the [workspaces](../resources/workspace.md) objects in the tenant.
+
+# [HTTP](#tab/http)
+<!-- {
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/place-list.md:245]: @slister1001 , I don't know how we missed this before, but can you address the following comments to all the new examples you added in this PR? 

I'm going to use this example 2 in place-list.md to describe what should be done:
There are lines like 244, 253 though 263, that you don't have to and shouldn't add; so please delete them. These lines are for the tabbed, language-specific code snippets that correspond to the REST API request. There's an automated weekly snippet update for all the new examples or modified examples that have been merged into main. As part of the weekly snippet update, any new examples, or existing examples that have been modified, would get 


Line 244: Delete th

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
@slister1001 , please address my comment about the lines you inserted to support upcoming code snippet for your new examples.

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
@slister1001 , it seems like you misunderstood my last comment about the code snippet lines. My comment applies to only the new examples you're adding to place-get.md, place-list.md, and place-update.md. My comment doesn't apply to the pre-existing examples, so please revert the changes to the pre-existing examples. If you don't do that, conceivably there can be a week or so lapse before the code snippet tool re-inserts the code snippets for various languages.

File: api-reference/beta/api/place-get.md [hunk 11]
```diff
@@ -66,16 +66,39 @@ If successful, this method returns a `200 OK` response code and the requested [p
 
 The following example specifies the **id** of a **room** to get its properties.
 
+# [HTTP](#tab/http)
 <!-- {
   "blockType": "request",
   "name": "get_room"
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/place-get.md:72]: After line 71:
FYI I inserted a sampleKeys property to specify the ID value 3162F1E1-C4C0-604B-51D8-91DA78989EB1 in your example request. It's something we recently started enforcing. It's for the Try-it experience. You can read up on sampleKeys in this topic: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211782/Set-up-for-API-Doctor-validation?anchor=json-element-descriptions

Comment (angelgolfer-ms) [reply api-reference/beta/api/place-get.md:72]: FYI I added sampleKeys preceding every example request in place-get.md and place-update.md that has an ID value.

**Review by angelgolfer-ms — APPROVED** [review]
Please see the comments I added as FYI. I'm setting this PR to content review complete. Please set the PR to ready to merge so that the PR will be published on the next work day.
